### ISSUE TITLE
Account

### ISSUE THREAD

**Issue by blueberrymuffin7708** [TheMagizz/DarkPremium#4]
it says my account Has been checkpoint

**Comment by UnderdogCadet** [comment]
yep, now you need to confirm your identity, if you are lucky enough its just that, perhaps this guy just stool your credentials.

**Comment by Brainy2** [comment]
Username And Passsword??
